modules/registry-change-management-state.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * registry/configuring-registry-storage/configuring-registry-storage-baremetal.adoc
+// * registry/configuring-registry-storage/configuring-registry-storage-vsphere.adoc
+
+[id="registry-change-management-state_{context}"]
+= Change image registry ManagementState
+
+To start the image registry, you must change `ManagementState` Image Registry Operator configuration from `Removed` to `Managed`.
+
+.Procedure
+
+////
+Stub for procedure.
```

Comment:
Is there a recommended procedure for changing the configuration from `Removed` to `Managed`?

Reply:
Do you mean what storage we should suggest? Eventually it'll be RHOCS, but today we don't have any preference, the customer can pick any storage we support: S3, Swift, NFS, etc.

Reply:
Thanks, @dmage!